Felienne/hedy
# [BUG FIX] 'error or empty' achievement can be achieved again

**Description**
This PR fixes an issue where we dependent on an unused variable in the TypeScript code for the 'error or empty' achievement check. Therefore it was never reached; and users where unable to achieve it. Thanks to an observant Hedy user (Kuno) this bug was noted.

**Fixes**
No related issue.

**How to test**
Make sure you have an account that doesn't have the 'error or empty' achievement yet. Create a program without an input but that is an actual program! For example, by using an `ask` without an `echo` in level 1. Verify that the achieved is received.
